src/components/Button/LinkButton.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { string } from 'prop-types';
+
+import SurveyButton from '.';
```

Comment:
this is odd, are you referencing `index` here, if so, I would explicitly say that it is `index` but ... in some case, you can get `circular` dependency here by mistake.